fenix/app/src/main/java/org/mozilla/fenix/debugsettings/navigation/DebugDrawerNavigation.kt

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.debugsettings.navigation
+
+import org.mozilla.fenix.R
+import org.mozilla.fenix.debugsettings.tabs.TabTools
+
+/**
+ * The navigation route for [DebugDrawerHome].
+ */
+const val DEBUG_DRAWER_ROUTE_HOME = "home"
```

Comment:
nit: I think we can also drop the `DEBUG_DRAWER...` as they are imported from `DebugDrawerNavigation`

Reply:
Since they're outside of a class, they would then show up in the global namespace as `ROUTE_HOME`. Obviously, this wouldn't be the case if we are able to long term modularize everything.

ALTERNATIVELY, we could do something like:
```
enum class DebugDrawerRoute(val route: String) {
    Home("home"),
    TabTools("tab_tools"),
}
```

and then we could have the string ID included here, too.

I'll try this with the next push

Reply:
Nice, this works well. I'm wondering if it can replace `DebugDrawerDestination` althogther?